tools/mutator-utils/mutator_helper.h

```diff
@@ -342,3 +342,21 @@ class ResizeIntegerHelper : public MutationHelper {
     updated = false;
   }
 };
+
+class UnaryInstHelper : public MutationHelper {
+  bool updated;
+public:
+ UnaryInstHelper(std::shared_ptr<FunctionMutator> mutator)
+      : MutationHelper(mutator), updated(false) {};
+  virtual void init() override;
+  virtual void reset() override { 
+    updated = false; 
+    }
```

Comment:
please be careful with indentation-- it's best to your `clang-format -i` on your code before issuing a pull request